### DISCUSSION TITLE
python highlighting differences in dark and light modes

### DISCUSSION THREAD

**Discussion by prajwel** [pradyunsg/furo#171] [Potential Issue] [answered]
Hi, 

Thank you for an amazing theme! I really love this.

Please see the following python snippets rendered in dark and light modes. 

![light_mode](https://user-images.githubusercontent.com/1685962/123762898-de5f6600-d8e0-11eb-98d3-0018b68eecf7.png)

![dark_mode](https://user-images.githubusercontent.com/1685962/123762929-e6b7a100-d8e0-11eb-9c3f-fda4311c8550.png)

The light mode is producing a grey font from the second line onwards.
Also, the string highlight colour also does not follow the format in the lines after the first one (this occurs in both dark and light modes). 

The RST source is given below.

```
.. code:: python

    >>> curvit.curve(events_list = 'AS1G06_084T01_9000000710uvtFIIPC00F1_l2ce.fits.gz', 
                     xp = 2636.71, yp = 907.91,
                     radius = 15,
                     bwidth = 50, 
                     background = 'auto')
```
page is at https://curvit.readthedocs.io/en/latest/getting_started.html

Thank you for your consideration.

**Comment by pradyunsg** [comment, marked as answer]
I think the sample code should be:

```
>>> import curvit
>>> curvit.makecurves(events_list = 'AS1G06_084T01_9000000710uvtNIIPC00F2_l2ce.fits.gz',
...                   threshold = 5)
```

instead of

```
>>> import curvit
>>> curvit.makecurves(events_list = 'AS1G06_084T01_9000000710uvtNIIPC00F2_l2ce.fits.gz',
                      threshold = 5)
```

(notice the 3 dots)

You're presenting it as a REPL prompts, and you're missing the continuation prompt in that output. Hopefully this helps. :)

**Reply by prajwel** [reply]
Thank you! That did help.
